src/Sarif.Sarifer/Rule.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    internal class Rule
```

Comment:
>Rule [](start = 19, length = 4)

Let's call it `SpamRule`. This isn't really a SARIF rule (SARIF rules are actually `ReportingDescriptor`s), and I don't want there to be confusion.